app/login/controller.js

```diff
@@ -0,0 +1,47 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import Auth0Lock from 'auth0-lock';
+
+const clientId = 'pVMVBa4Xdu2xiOZFI6EMfSEq9PNNdVYD';
+const domain = 'dev-9n0-z7z7.us.auth0.com';
```

Comment:
To avoid security problem consider to install [ember-cli-dotenv](https://github.com/fivetanley/ember-cli-dotenv) and move this keys to `.env` file (and not push it into repo, is risky!).
Name them like `AUTH0_CLIENT_ID` and `AUTH0_DOMAIN`

I did it in this PR, take a look when you have some problems - https://github.com/igorjuraszek/zadanie-II-ember/pull/4/files